### DISCUSSION TITLE
How to add windows Peer?

### DISCUSSION THREAD

**Discussion by mihailigoshin** [netbirdio/netbird#690] [Q&A / Support]
I have a private organization and with Linux it's really easy with the key but for Windows just creates a new organization even though that user has been invited.

**Comment by mlsmaycon** [comment]
@mihailigoshin, just to understand better the issue with the new organizations,  are you running on self-hosted or cloud?

on windows you can also use the setup key, the installer adds both UI and CLI clients, and you can use the same setup-key flag in PowerShell.
